components/collections/codepointtrie_builder/cpp/list_to_ucptrie.cpp

```diff
@@ -16,37 +16,11 @@
  * - standard output is the build UCPTrie as a TOML file
  */
 
-const int32_t TRIE_SMALL = 0;
-const int32_t TRIE_FAST = 1;
 extern "C" {
 int construct_ucptrie(const int32_t defaultValue, const int32_t errorValue,
-                      const int32_t trieTypeInt, const int32_t valueWidthInt,
+                      const UCPTrieType trieType,
```

Comment:
I think enums on C ABI are brittle because they don't have a guaranteed representation, so we should keep these as ints. `native.rs` does this as well.

Reply:
Yes, that can be an explicit cast

Reply:
Done.